cmd/antrea-agent/agent.go

```diff
@@ -177,6 +194,12 @@ func run(o *Options) error {
 	// if AntreaPolicy feature is enabled.
 	statusManagerEnabled := antreaPolicyEnabled
 	loggingEnabled := antreaPolicyEnabled
+	var denyConnStore connections.ConnectionStore
+
+	if features.DefaultFeatureGate.Enabled(features.FlowExporter) {
+		connStore := connections.NewConnectionStore(ifaceStore, proxier, true)
+		denyConnStore = connections.NewDenyConnectionStore(connStore)
```

Comment:
It is better to move the NewConnectionStore into the NewDenyConnectionStore function.
Same with NewConntrackConnectionStore below.